### DISCUSSION TITLE
how to create repository

### DISCUSSION THREAD

**Discussion by ghost** [skills/.github#237] [General]
how to create repository

**Comment by Kenastark** [comment]
To create a new repository (also called 'repo'), follow the steps below.
1. log into your GitHub profile. It will have the URL https://github.com/your_username
2. Click on **Repositories** at the top
3. You should new a green button🟩 to your right with the label **New**. Click on it
4. Fill in your Repository name.
5. Add a description (optional)
6. Choose whether it will be a publicly (Public) available repo or a Private one only available for you to see
7. Add a README file
8. Finally click on **Create repository**

**Reply by Learn5Git** [reply]
Good to see other people in this course. I am on Module 2 (How to migrate your repository). Does anyone have the steps for how to migrate your repository for this course module? I think the course link is outdated--there is a 404 page if you click on the link for that course name.

**Reply by Kenastark** [reply]
1. I think you would first need to create your reposition using the steps above.

2. Then you click on the newly created repo.

3. Should should then see a **"Add File button"**

<img width="1006" alt="Screenshot 2023-06-05 at 09 35 20" src="https://github.com/skills/.github/assets/77270835/13977d1c-2859-4097-8b5f-a28107b337e5">

4. Click on "Upload files"

5. Then drag and drop or select your file 

6. Click the green button🟩 below with the label "Commit changes"

PS: don't forget to add a meaningful commit message✌🏾
